test/orangeade_test.exs

```diff
@@ -11,4 +13,17 @@ defmodule OrangeadeTest do
       stream: 1
     ]
   end
+
+  test "the boundnatural generator should be uniform" do
+    boundnaturals = Caffeine.Stream.take(BoundNatural.stream(limit: 100), 100)
+    epsilon = 5
```

Comment:
The "epsilon" is great because it formally captures what you're doing. You can do it for the other test cases as well. It should be possible to abstract the relationship between `Stream.take`ing 100 and the `epsilon = 5`. I'm not sure how clean that'd be though.